defi/src/adaptors/handlers/storeAdaptorData/index.ts

```diff
@@ -176,19 +190,11 @@ export const handler2 = async (event: IStoreAdaptorDataHandlerEvent) => {
       // Import adaptor
       const adaptor: Adapter = (await importModule(module)).default;
       // if an adaptor is expensive and no timestamp is provided, we try to avoid running every hour, but only from 21:55 to 01:55
-      if (!isRunFromRefillScript && adaptor.isExpensiveAdapter && !isTimestampProvided) {
-        const date = new Date(currentTimestamp * 1000)
-        const hours = date.getUTCHours()
-        if (hours > 5) {
-          console.info(`[${adapterType}] - ${index + 1}/${protocols.length} - ${protocol.module} - skipping because it's an expensive adapter and it's not the right time`)
-          return
-        }
-      }
-      const adapterVersion = adaptor.version
-      const isVersion2 = adapterVersion === 2
-      const v1Timestamp = (timestamp !== undefined ? toTimestamp : fromTimestamp)
-      const endTimestamp = (isVersion2 && !timestamp) ? LAMBDA_TIMESTAMP : toTimestamp // if version 2 and no timestamp, use current time as input for running the adapter
-      let recordTimestamp = isVersion2 ? toTimestamp : v1Timestamp // if version 2, store the record at with timestamp end of range, else store at start of range
+      const adapterVersion = adaptor.version ?? 1
+      const isAdapterVersionV1 = adapterVersion !== 2
+
+      let endTimestamp = toTimestamp
+      let recordTimestamp = toTimestamp
       if (isRunFromRefillScript) recordTimestamp = fromTimestamp // when we are storing data, irrespective of version, store at start timestamp while running from refill script? 
```

Comment:
uhhh how does this make sense?

Reply:
ddb stores data with SK field = record field, but in postgres, I felt storing daily record tied to date string instead was better, this way we dont end up with two records for the same day.

But we determine the date based on time range, for v1 adapters,  start & end timestamp fall on the same day (or maybe one second into the next day?) to be safe, I use the start timestamp for v1 adapters

v2 adapter, the range is past 24 hours, so I use end timestamp to compute record date string.